lstchain/mc/tests/test_mc.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+from numpy.testing import assert_allclose
```

Comment:
if you already import `numpy`, I'd suggest using directly `np.testing.assert_allclose` in the test.